src/System.Management.Automation/help/UpdatableHelpSystem.cs

```diff
@@ -1682,7 +1683,7 @@ internal static void SetDisablePromptToUpdateHelp()
         {
             try
             {
-                ConfigPropertyAccessor.Instance.SetDisablePromptToUpdateHelp(true);
+                PowerShellConfig.Instance.SetDisablePromptToUpdateHelp(true);
```

Comment:
We have plans to make Help System optional. So it seems Config API should be public in future.

Reply:
The config APIs needs more improvements for sure, one thing is is the singleton pattern really needed -- can the methods just be static methods?

Reply:
Yes. we can revisit the API later. I think we could create a common API to re-use the same config files and registry for customer modules.

Reply:
I would rather to not re-use the same config file for modules. Synchronizing the updating of the config file among processes is expensive.